### ISSUE TITLE
Remove a dark background to the bottom layer if the modal contains .coinbaseOverlay

### ISSUE THREAD

**Issue by luixxiul** [brave/browser-laptop#6093]
**Describe the issue you encountered:** https://github.com/brave/browser-laptop/pull/4820#issuecomment-265751399

Add `.last` to the modal overlay _unless_ the modal contains `.coinbaseOverlay`. The iframe inside `.coinbaseOverlay` itself has the dark background.

- Brave Version: macOS

- Any related issues:

CC @willy-b

**Comment by willy-b** [comment]
Nice. You can assign this to me if you want and if it can wait until after PRs #5809 and #6039 (targeted today).

**Comment by mrose17** [comment]
@willy-b - any luck?

**Comment by willy-b** [comment]
was going to do this after #6039 but that is still outstanding. I'm actually out of cycles for a while now, you should probably reassign this.

**Comment by mrose17** [comment]
@willy-b - understood, thanks and no worries!

**Comment by luixxiul** [comment]
I tried and I came to a conclusion that the background layer of coinbase iframe widget should be removed instead of our modal overlay. If we remove the background of modal overlay, lag occurs after clicking the button until the coinbase widget is actually rendered.

**Comment by mrose17** [comment]
i'm going to mark this as "won't fix"... for the reasons above (and awaiting the new addfunds service).
